### DISCUSSION TITLE
[Feature Request]: 海报上的TMDB、豆瓣、bangumi不够明显，我这边做了几个看是否够用

### DISCUSSION THREAD

**Discussion by Kev1nT** [jxxghp/MoviePilot#3489] [讨论]
### 当前程序版本

1.8.8

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

海报上的TMDB、豆瓣、bangumi不够明显，我这边做了几个看是否够用
![PixPin_2024-05-15_14-00-18](https://github.com/jxxghp/MoviePilot/assets/49979564/34673c45-e82d-43c9-95f1-e9a797796873)

### 参考资料

![组 1](https://github.com/jxxghp/MoviePilot/assets/49979564/914de0c1-3151-44a3-84d9-eff995926198)
![豆瓣-iOS-512x512](https://github.com/jxxghp/MoviePilot/assets/49979564/9f4a14d8-ce27-446a-83c0-a4b23fa87f04)
![Bangumi 番组计划-iOS-512x512](https://github.com/jxxghp/MoviePilot/assets/49979564/658f7270-4de4-4c44-85ef-f7b86e83c92d)

**Comment by Kev1nT** [comment]
如果在同色海报上不够明显的话，可以手工加一点投影即可~

**Comment by jxxghp** [comment]
效果如图，貌似不太自然：

![image](https://github.com/jxxghp/MoviePilot/assets/51039935/1da64695-7087-45d6-a336-78a09302f2f0)

**Comment by Kev1nT** [comment]
> 效果如图，貌似不太自然：
> 
> ![image](https://private-user-images.githubusercontent.com/51039935/330780077-1da64695-7087-45d6-a336-78a09302f2f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4MzEyMTYsIm5iZiI6MTcxNTgzMDkxNiwicGF0aCI6Ii81MTAzOTkzNS8zMzA3ODAwNzctMWRhNjQ2OTUtNzA4Ny00NWQ2LWEzMzYtNzhhMDkzMDJmMmYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDAzNDE1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkMzRmZjlmMTJlYjM3MzI4ZmM0NjhlOGY2OWE0YjZmZTdlODFmMjAzOTMwOWY3N2MzZDZkMzA1MmUyYmYwZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UtEqbijPGtxDqCebJCHymXOoGU7liLCAQktiKzIU_O8)



![Bangumi 番组计划-iOS-512x512](https://github.com/jxxghp/MoviePilot/assets/49979564/cf637260-1265-4773-9d6f-4dcb175e03b6)
![豆瓣-iOS-512x512](https://github.com/jxxghp/MoviePilot/assets/49979564/e4c1a801-f8b7-4fec-a75a-7ca2a4a46e3b)
![组 1](https://github.com/jxxghp/MoviePilot/assets/49979564/e3752bf8-9201-4d4c-ad06-efbc99543aab)

![组 1](https://github.com/jxxghp/MoviePilot/assets/49979564/63da4bb3-2066-4759-acca-112bae43675d)

**Comment by pogame** [comment]
![tmdb1](https://github.com/jxxghp/MoviePilot/assets/53455682/ab153fd2-5c47-4a57-bb15-11500f39a354)
用ps改了个tmdb的标出来，感觉比目前原版自带的舒服，效果如下
![image](https://github.com/jxxghp/MoviePilot/assets/53455682/380b40a9-b28c-499c-be6c-702edb5197cc)
